molo/core/models.py

```diff
@@ -41,6 +41,12 @@ class SectionPage(Page):
         index.SearchField('description'),
     )
 
+    @property
```

Comment:
Why make this a property when it doesn't really make a difference in the templates?